include/range/v3/range_access.hpp

```diff
@@ -247,6 +255,12 @@ namespace ranges
             (
                 pos.next()
             )
+            template<typename Cur>
+            static RANGES_CXX14_CONSTEXPR auto post_increment(Cur & pos)
+            RANGES_DECLTYPE_AUTO_RETURN
+            (
+                pos.post_increment()
+            )
```

Comment:
I gotta say that I hate that the name of the customization points are `next()` and `post_increment()`. No symmetry. I actually prefer `next()` and `next(int)`, even though the `int` hack is not necessary here.